src/db.rs

```diff
@@ -144,20 +147,26 @@ impl DbInner {
         &self,
         sst: &SsTableHandle,
         key: &[u8],
+        key_hash: u64,
     ) -> Result<bool, SlateDBError> {
         if !sst.range_covers_key(key) {
             return Ok(false);
         }
         if let Some(filter) = self.table_store.read_filter(sst).await? {
-            return Ok(filter.has_key(key));
+            return Ok(filter.has_key(key_hash));
         }
         Ok(true)
     }
 
-    async fn sr_may_include_key(&self, sr: &SortedRun, key: &[u8]) -> Result<bool, SlateDBError> {
+    async fn sr_may_include_key(
+        &self,
+        sr: &SortedRun,
+        key: &[u8],
+        key_hash: u64,
```

Comment:
Nit: WDYT about Option<u64>, and call filter::hash_key() if it's none?

Reply:
Nit: Mind adding rustdocs to this method to clarify why we have both `key` and `key_hash` args?

Reply:
The reason is as above.